app/src/test/java/org/simple/clinic/editpatient/deletepatient/DeletePatientUpdateTest.kt

```diff
@@ -64,4 +64,30 @@ class DeletePatientUpdateTest {
             hasEffects(ShowConfirmDiedDialog(patientName = patientName) as DeletePatientEffect)
         ))
   }
+
+  @Test
+  fun `show home screen after patient is deleted`() {
+    val model = defaultModel.patientNameLoaded(patientName)
+
+    updateSpec
+        .given(model)
+        .whenEvent(PatientDeleted)
+        .then(assertThatNext(
+            hasNoModel(),
+            hasEffects(ShowHomeScreen as DeletePatientEffect)
+        ))
+  }
+
+  @Test
+  fun `show home screen after patient is marked as dead`() {
```

Comment:
Can we rename these tests to be along the lines of `when <situation>, then <outcome>`?